community/modules/scheduler/schedmd-slurm-gcp-v5-controller/README.md

```diff
@@ -11,13 +11,14 @@ The [user guide][slurm-ug] provides detailed instructions on customizing and
 enhancing the Slurm on GCP cluster as well as recommendations on configuring the
 controller for optimal performance at different scales.
 
-> **_WARNING:_** The variables [enable\_reconfigure],
-> [enable\_cleanup\_compute] and [enable\_cleanup\_subscriptions], if set to
-> true, require additional dependencies **to be installed on the system running
-> `terraform apply`**. Python3 (>=3.6.0, <4.0.0) must be installed along with
-> the pip packages listed in the [requirements.txt] file of
-> [SchedMD/slurm-gcp]. See the
-> [documentation below](#live-cluster-reconfiguration-enable_reconfigure).
+> **Warning**: The variables `enable_reconfigure`,
```

Comment:
It also just occurred to me that you may need to define the default gcloud project by running `gcloud config set project <project_id>`. If you don't do that, it will fail in weird ways.

Reply:
This should be now resolved. I will make sure we validate that.